src/Traits/ExecuteCommand.php

```diff
@@ -45,7 +45,7 @@ protected function getCommandArrays(string $type = 'install'): array
     {
         $commands = [
             'install' => [
-                'npm' => 'npm install',
+                'npm' => 'npm ci',
```

Comment:
We need to do a clean install here as well. Otherwise changes to electron-plugin wont come through.

For example when running `npm run build:watch` from the plugin directory & running `artisan native:serve` in a different terminal session.

Having to clean install all dependencies might slow things down a little bit. This is a tradeoff but it could be that the cache negates this. Did not benchmark this yet

Reply:
It would be great if we didn't have to `ci` but could rather just force the `electron-plugin` to be reinstalled from scratch...